scripts/gtcadis.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+import argparse
+import yaml
+import io
+
+import numpy as np
+from pyne.mesh import Mesh
+from pyne.partisn import write_partisn_input, isotropic_vol_source
+from pyne.dagmc import discretize_geom, load
+from pyne import nucname
+from pyne.bins import pointwise_collapse
+
+
+config_filename = 'config.yml'
+
+config = \
+    """
+# Optional step to assess all materials in geometry for compatibility with
+# SNILB criteria
+step0:
+
+# Prepare PARTISN input file for adjoint photon transport
+step1:
```

Comment:
I guess step1: should be 'step1': as it is a nested collection.

Reply:
I am not sure what you mean @moatazharb ?  Do you mean it should have quotes?